meta-rdk-mtk-bpir4/recipes-ccsp/rdk-wifi-libhostap/rdk-wifi-libhostap_2.11.bb

```diff
@@ -0,0 +1,257 @@
+SUMMARY = "RDK-WiFi-LIBHOSTAP for RDK CcspWiFiAgent components"
+SUMMARY = "This recipe compiles and installs the Opensource hostapd as a dynamic library for RDK hostap authenticator"
+SECTION = "base"
+LICENSE = "BSD-3-Clause"
+
+FILESEXTRAPATHS_prepend:="${THISDIR}/files:"
+PROVIDES = "rdk-wifi-libhostap"
+RPROVIDES_${PN} = "rdk-wifi-libhostap"
+DEPENDS += "libnl openssl"
+
+DEPENDS_append = " ucode"
+
+inherit autotools pkgconfig
+
+def getowe_defined(d):
+    if d.getVar('MACHINE_IMAGE_NAME', True) in [ 'CGM4981COM' ]:
+        return '-DCONFIG_OWE'
+    else:
+        return ''
+
+def get_hostapd_pv(d):
+    ret_val = '2.9' #HOSTAPD=DEFAULT
+    distro_features = d.getVar('DISTRO_FEATURES', True)
+    if distro_features:
+        val = distro_features.split('HOSTAPD_')
+        if len(val) > 1:
+            ret_val = val[1].split()[0].replace('_','.')
+    return ret_val
+
+HOSTAPD_PV = "${@get_hostapd_pv(d)}"
+#true for 2.9 and 2.10
+PRIOR_BUILD = "${@'true' if d.getVar('HOSTAPD_PV', True) == '2.9' or d.getVar('HOSTAPD_PV', True) == '2.10' else 'false'}"
+
+SRC_URI = "git://w1.fi/hostap.git;protocol=https;branch=main;destsuffix=${S}/source/hostap-${HOSTAPD_PV};name=${HOSTAPD_PV}"
+SRCREV_2.9 = "7cb441432c4379b92ef044059fa21eec7e3bb355"
+SRCREV_2.10 = "9d07b9447e76059a2ddef2a879c57d0934634188"
```

Comment:
Why are 2.9 and 2.10 needed here?
Remove all the support for 2.9 and 2.10 to make the recipe easier to read.